Detectors/EMCAL/simulation/src/RawOutputPageHandler.cxx

```diff
@@ -0,0 +1,85 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <fmt/format.h>
+#include <Headers/RAWDataHeader.h>
+#include <EMCALSimulation/RawOutputPageHandler.h>
+
+using namespace o2::emcal;
+
+RawOutputPageHandler::RawOutputPageHandler(const char* rawfilename, o2::InteractionRecord firstIR) : mTimeframeHandler(firstIR), mOutputStream(rawfilename)
```

Comment:
Why do you set the IR of the 1st TF start to the IR of your 1st trigger? The TF start must be common for all detectors. 
You better don't touch ``HBFUtils::getFirstIR`` to avoid de-synchronization with others dets., later I may convert HBFUtils to singleton with centrally set 1st IR (via ConfigurableParam).

Reply:
OK, then the IR must be set from outside and can no longer be obtained from EMCAL-only data. How do other detectors do this?

As EMCAL will not receive HBF triggers I only use the HBF utils to marks start/stop of timeframe.

Reply:
Yes, but the TFs still have to be in sync with other detectors, so you should rely on the HBFUtils::getFirstIR set externally.

Reply:
So this means I should rely on having the HBFUtils owned and configured externally?

Reply:
Yes, the 1st BC/orbit will be either extracted from the GRP or read from the common configuraton file (at the moment just default 0/0 is used).
in any case particular workflows should not modify it to avoid desynchronization.

Reply:
OK, then I take the HBFUtils as a pointer to a const object and expect them to be set from outside (in a dedicated component).